pageserver/src/storage_sync2.rs

```diff
@@ -498,6 +498,18 @@ impl RemoteTimelineClient {
         self.metrics.remote_physical_size_gauge().set(size);
     }
 
+    pub fn latest_files_total_size(&self) -> Result<u64, anyhow::Error> {
```

Comment:
Few times already I've looked at this, and then reminded myself of what the latest_files are. Since this seems to be only called from the `Tenant::get_remote_size` I wonder if this could be `get_timeline_remote_size`, just to have some sort of ... symmetry?

It will probably need to be changed if for example the remote client becomes shared between multiple timelines or something like that.

Reply:
fixed. and also reused already existing metric instead of calculating it